src/main/kotlin/bsu/cc/views/MainView.kt

```diff
@@ -44,34 +60,66 @@ class MainView : View("Conflict Checker") {
         center {
             vbox {
                 addClass(Styles.content)
-                button("Click me") {
-                    setOnAction {
-                        alert(INFORMATION, "Well done!", "You clicked me!")
+                borderpane {
+                    left {
+                        button("Choose File") {
+                            setOnAction {
+                                if(("" + fileNameField.text().text).trim().length < 1) {
```

Comment:
Any reason for doing string concatenation instead of (assuming text is not a string)
`fileNameField.text().text.toString().trim().length`

Reply:
Thanks for pointing that out.  That's a remnant of when I didn't realize .text() didn't return a string.  CHanged it to 'fileNameField.text().text.trim().length' (.text().text does return a string)